### ISSUE TITLE
Why does this application have "Webpage Commander"?

### ISSUE THREAD

**Issue by jathek** [kangyu-california/PersistentWindows#314]
This seems like a feature that is completely unrelated to PersistentWindows. It should be a separate application entirely.

**Comment by kangyu-california** [comment]
The webpage commander feature relies on existing PW infrastructure, it seems more natural for me to integrate it into PW rather than creating another standalone app.

Although the new feature seems to deviate from the original goal of PW,  all these features serve the same purpose of improving user experience on Windows.

**Comment by jathek** [comment]
You are bundling in a feature that is not related at all to PersistentWindows. People install this specifically for the sole feature of remembering window positions. Imo, if you want browser shortcuts, you should write a chrome extension for yourself instead of bloating this software.

**Comment by kangyu-california** [comment]
@jathek 
I see your point. 
The basic functionality of PersistentWindows is pretty much mature now.

Users who want only the basic function may stick to 5.53 by running it with command line -check_upgrade=0

**Comment by jathek** [comment]
That is not really fair to people. You already have included fixes for window restoration in later versions. I get that you are the owner of the repo, but it seems like you are forcing in a feature for yourself that is completely unrelated to the history of the project.

And what are you going to do with different browsers? I use Firefox and most of the buttons in Webpage Commander don't even work. Are you willing to support different browsers, or will anyone using something other than Chrome have to put up with a broken feature?

**Comment by kangyu-california** [comment]
webpage commander works by translating command shortcut, as demonstrated in the following code snippet
![image](https://github.com/kangyu-california/PersistentWindows/assets/59128756/db64e676-e438-45e1-b2d1-c0a3061efd66)

The webpage command feature is only activated upon user pressing Alt + W hotkey or selected from menu, moreover the hotkey can be  turned off via command line  (#316).

As the owner of this PW repo which has around 100K users, I am fully aware of the importance of keeping user privacy intact. Please read the privacy statement in readme if you have any concerns.

To my knowledge every newly introduced feature is well documented in README, HELP and Release note. Everything is open sourced which means user can censor and make their own judgement, and finally the code is free to fork and modify.

**Comment by EarMaster** [comment]
I agree with OP that this should not be part of this tool. It has nothing to do with Window management. Of course you can do what you want with the tool you are maintaining, but please don't assume that this is a feature many of your users are interested in.

**Comment by TristanYoung** [comment]
I like the idea of PersistentWindows.

Unfortunately you've added something that falls outside the scope of Window management, and then recommend people stay on an earlier version if they prefer.

I do not subscribe to the idea that helpful utilities need to be encumbered by any code that doesn't serve the original purpose of the utility, nor having to remain on an older/outdated version in order to avoid something that shouldn't be included in the first place.

I'd rather have seen an installer for those people who are not computer-savvy, and proper support for auto-starting the utility, as this would help a larger swath of prospective users.  Adding those would not break with the scope of PersistentWindows, as they directly impact [novice users that desire the CORE functionality] in a positive-benefit way, and actually encourage/permit them to use the utility.

I'm going with another solution for myself, but will keep an eye on this one.  As soon as this utility is cleaned up, presenting features entirely within the scope of the project concept and none other, I'd be happy to recommend it to others.

**Comment by kangyu-california** [comment]
@all

Developing side-features like Webpage commander while maintaining PW core-features actually helped me to keep energetic in the last two years.
I believe it is good timing to strip Webpage Commander from next PW release. 

Thank you all for the patience.
